crawl-ref/source/mon-info-flag-name.h

```diff
@@ -55,9 +54,18 @@ static const vector<monster_info_flag_name> monster_info_flag_names = {
     { MB_RESISTANCE, "resistant", "unusually resistant", "resistant"},
     { MB_INVISIBLE, "invisible", "slightly transparent", "invisible"},
     { MB_REGENERATION, "regenerating", "regenerating", "regenerating"},
-    { MB_RAISED_MR, "resisting magic", "resistaant to hostile enchantments", "resisting magic"},
+    { MB_RAISED_MR, "resisting magic", "resistant to hostile enchantments", "resisting magic"},
     { MB_INJURY_BOND, "sheltered", "sheltered from injuries", "sheltered"},
     { MB_GOZAG_INCITED, "incited", "incited by Gozag", "incited"},
+    { MB_CLOUD_RING_THUNDER, "", "surrounded by a ring of thunder", "" },
+    { MB_CLOUD_RING_FLAMES, "", "surrounded by a ring of flames", "" },
+    { MB_CLOUD_RING_CHAOS, "", "surrounded by a ring of chaotic energy", "" },
+    { MB_CLOUD_RING_MUTATION, "", "surrounded by a ring of mutagenic energy", "" },
+    { MB_CLOUD_RING_FOG, "", "surrounded by a ring of fog", "" },
+    { MB_CLOUD_RING_ICE, "", "surrounded by a ring of freezing clouds", "" },
+    { MB_CLOUD_RING_DRAINING, "", "surrounded by a ring of negative energy", "" },
+    { MB_CLOUD_RING_ACID, "", "surrounded by a ring of acid", "" },
+    { MB_CLOUD_RING_MIASMA, "", "surrounded by a ring of miasma", "" },
```

Comment:
I think that if I type this suggestion I can then use the web UI to commit it? Just some display tweaks so that this appears in the HUD and doesn't take too much space in the targeter/^x.

```suggestion
    { MB_CLOUD_RING_THUNDER, "clouds", "surrounded by thunder", "clouds" },
    { MB_CLOUD_RING_FLAMES, "clouds", "surrounded by flames", "clouds" },
    { MB_CLOUD_RING_CHAOS, "clouds", "surrounded by chaotic energy", "clouds" },
    { MB_CLOUD_RING_MUTATION, "clouds", "surrounded by mutagenic energy", "clouds" },
    { MB_CLOUD_RING_FOG, "clouds", "surrounded by fog", "clouds" },
    { MB_CLOUD_RING_ICE, "clouds", "surrounded by freezing clouds", "clouds" },
    { MB_CLOUD_RING_DRAINING, "clouds", "surrounded by negative energy", "clouds" },
    { MB_CLOUD_RING_ACID, "clouds", "surrounded by acidic fog", "clouds" },
    { MB_CLOUD_RING_MIASMA, "clouds", "surrounded by foul miasma", "clouds" },
```